crates/paimon/src/error.rs

```diff
@@ -23,11 +23,11 @@ pub type Result<T, E = Error> = std::result::Result<T, E>;
 /// Error type for paimon.
 #[derive(Debug, Snafu)]
 pub enum Error {
-    #[snafu(display("Paimon data invalid for {}: {:?}", message, source))]
+    #[snafu(display("Paimon data invalid for {}", message))]
```

Comment:
Is this change relevant?

Reply:
The changes here are to adapt to error handling. I will deal with it carefully later to ensure the minimum scope of changes.